Core/Core/Configuration/Configurable/Configurable.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  Configurable.swift
+//  Core
+//
+//  Created by Eugene Yatsenko on 27.10.2023.
+//
+
+import Foundation
+import FirebaseCore
+import Swinject
+
+public enum Apps {
+    case openEdx
+    case edX
+}
```

Comment:
@eyatsenkoperpetio What are we trying to achieve here by creating different enums?